lighthouse-core/gather/drivers/driver.js

```diff
@@ -596,13 +596,19 @@ function captureJSCallUsage(funcRef, set) {
   const originalPrepareStackTrace = Error.prepareStackTrace;
 
   return function() {
+    const args = Object.values(arguments); // callee's arguments.
```

Comment:
it's actually fine, because this runs in Chrome via `evaluateAsync`. Maybe we should mark the function better that it only runs inside the target page?

`Object.values` *is* weird with `arguments`, though. Maybe `Array.from`?